python/makeALPACAEvents.py

```diff
@@ -1,12 +1,11 @@
+#!/usr/bin/env python
+from __future__ import print_function
+from __future__ import division
 import os,sys,getopt
 import numpy,math
 import ROOT
-
-hGeV = 6.58211928*pow(10.,-16)* pow(10.,-9) # no units or it messes up!!
-c_light = 2.99792458e+10
-mres="1.0d0"
-gax="1d-7"
-nev="100"
+hGeV      = 6.58211928*pow(10.,-16)* pow(10.,-9) # no units or it messes up!!
```

Comment:
Why not `6.58211928e-27`? Does that give the same results? If not, can you use `pow(10.,-27)`?

Reply:
Also, please don't change the formatting like this. There should be exactly one space before and after `=`. This and the following line are changed for no reason.

Reply:
what about using >>> ROOT.TMath.C()
299792458.0
>>> ROOT.TMath.H()
6.62606876e-34

if everybody hardcodes its own values, will be a mess.

Reply:
I agree: Would be much nicer to not hard-code these!

Are they already in `shipunit`?

Reply:
okay, agreed.
Then, should I  change this [line](https://github.com/ShipSoft/FairShip/blob/master/python/shipunit.py#L271)?

Reply:
> okay, agreed.
> Then, should I change this [line](https://github.com/ShipSoft/FairShip/blob/master/python/shipunit.py#L271)?

I would say yes (and `c_light` and `pi` as well).

But we might want to consider using <https://github.com/scikit-hep/hepunits> rather than implementing all of these ourselves. The package is actively maintained by the HEP python community (including but not limited to several LHCb authors and is in my experience very reliable).

Reply:
I am using the defn from shipunit now.